### ISSUE TITLE
Caseless option for context and list keywords

### ISSUE THREAD

**Issue by AlexGrok** [mpcjanssen/simpletask-android#829]
May this option be implemented? In simplest way just to ignore case for such words when creating list of such words. If You will implement this, than filter list looks more accurate and show only one keyword for +Home and +home items. Thanks!

**Comment by smichel17** [comment]
There is a `Sort case sensitive` option in the filter that you can uncheck. Is that what you would like or is there something else?

**Comment by AlexGrok** [comment]
No, I mean not taking into account case of keywords for quick filters.

**Comment by mpcjanssen** [comment]
Can already be achieved using Lua filtering and sort. Would be difficult to integrate into the UI.

**Comment by smichel17** [comment]
I don't think it can quite be achieved using lua, since @AlexGrok is looking to have this take place in the *quick filter drawer* (left), not on sort, display, etc.

I'd imagine this would be implemented as a global setting which would affect task parsing (ie, `+home` would be parsed the same as `+Home`)

**Comment by mpcjanssen** [comment]
@AlexGrok I think this is finally fixed in ebffbde697d9a42ac2f3fb7a3e2689ee54ca1d7f .
Before the case sensitive option was not consistently applied.

**Comment by mpcjanssen** [comment]
Seems I misunderstood again what is requested here. This could be achieved by making the tag and context filters case insensitive. Or always lowercase tags. Or a combination of both.
